packages/www/components/Marketing/JobApplicationForm/index.tsx

```diff
@@ -0,0 +1,437 @@
+                  size="3"
+                  id="lastName"
+                  css={{ width: "100%", mb: "$3" }}
+                  name="lastName"
+                  type="text"
+                  placeholder="Last name"
+                  required={name === "required"}
+                  value={lastName}
+                  onChange={(e) => setLastName(e.target.value)}
+                />
+              </Grid>
+            )}
+            {phone !== "off" && (
+              <TextField
+                size="3"
+                id="phone"
+                css={{
+                  width: "100%",
+                  mb: "$3",
+                  mx: "$2",
+                  "@bp1": {
+                    mx: "$4",
+                  },
+                }}
+                name="phone"
+                type="phone"
+                placeholder="Phone (optional)"
+                required={phone === "required"}
+                value={phoneNumber}
+                onChange={(e) => setPhoneNumber(e.target.value)}
+              />
+            )}
+
+            <TextField
+              size="3"
+              id="email"
+              css={{
+                width: "100%",
+                mb: "$3",
+                mx: "$2",
+                "@bp1": {
+                  mx: "$4",
+                },
+              }}
+              name="email"
+              type="email"
+              placeholder="Email"
+              required
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+            />
+
+            {resume !== "off" && (
+              <Box
+                css={{
+                  mb: "$3",
+                  width: "100%",
+                }}>
+                <Box
+                  css={{
+                    width: "100%",
+                    cursor: "pointer",
+                    p: "$1",
+                    mb: "$0",
+                    height: "auto",
+                    border: "1px solid $colors$mauve7",
+                    borderRadius: "$1",
+                  }}
+                  {...getRootProps({ style })}>
+                  <Box as="input" {...getInputProps()} />
+                  <Box
+                    as="p"
+                    css={{
+                      width: "100%",
+                      height: "100%",
+                      border: "1px dotted $colors$mauve7",
+                      borderRadius: "$1",
+                      m: 0,
+                      fontSize: "$3",
+                      p: "$3",
+                      transition: "border .24s ease-in-out",
+                      minWidth: "296px",
+                      minHeight: "70px",
+                      display: "flex",
+                      justifyContent: "center",
+                      alignItems: "center",
+                    }}>
+                    {loadingPdf
+                      ? "Uploading..."
+                      : "Drag and Drop your CV file or upload files here"}
```

Comment:
Change text to "Drag and Drop your CV file or upload file here" (file should be singular).